docs/src/ref_impl/installer.md

```diff
@@ -0,0 +1,73 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Installer
+
+## Configuring And Building Installer For Ghaf
+
+You can obtain the installation image for your Ghaf configuration using this guide. To check possible configuration options see (ghaf documentation for modules options)[https://tiiuae.github.io/ghaf/ref_impl/modules_options.html#ghafinstaller].
```

Comment:
Replace with: (pay attention to the link)

`You can obtain the installation image for your Ghaf configuration. To check possible configuration options, see [Modules Options](../ref_impl/modules_options.md).`